### PR TITLE
DR secondary generate operation token styling updates

### PR DESCRIPTION
**Pull request by chelshaw**
This PR updates the styling and content on the DR Secondary Generate Operation Token modal to better match designs. Updates below:

generate-1

generate-2

### CODE REVIEW

File: ui/app/templates/components/shamir-modal-flow.hbs [hunk 1]
```diff
@@ -131,24 +146,24 @@
...
                   <HoverCopyButton @copyValue={{otp}} />
                   <code class="is-word-break">{{otp}}</code>
                 </div>
               </div>
-              <div class="has-text-grey is-size-8 has-bottom-margin-l">
-                This OTP will be used to encode the generated operation token. <span class="has-text-semibold">Save this</span>, as you will need it later to decode the operation token.
-              </div>
             {{/if}}
             <p>
               Generate an operation token by entering a portion of the master key. Once all portions are entered, the generated token may be used to manage your secondary Disaster Recovery cluster.
```
Comment (noelledaley) [inline ui/app/templates/components/shamir-modal-flow.hbs:163]: nit: we should specify that this needs to be a portion of the primary's master key.

File: ui/app/templates/components/shamir-modal-flow.hbs [hunk 2]
```diff
@@ -131,24 +146,24 @@
...
           <div class="box is-shadowless is-marginless no-padding-top is-fullwidth no-padding-sides" data-test-form-text>
             {{#if otp}}
-              <div>
-                <h4 class="title is-7 has-bottom-padding-m is-fullwidth">
+              <div class="has-bottom-margin-xxl">
+                <h4 class="field-title">
                   One-time password (OTP)
                 </h4>
+                <p class="help has-text-grey has-bottom-margin-xs">
+                  This OTP will be used to encode the generated operation token. <span class="has-text-semibold">Save this</span>, as you will need it later to decode the operation token.
```
Comment (noelledaley) [inline ui/app/templates/components/shamir-modal-flow.hbs:154]: nit: technically, this should be "decode the generated operation token", just like the instructions here: https://github.com/hashicorp/vault/pull/9355/files#diff-0736a8f910677faecdbf8bfb0178ed3eR32

**Comment by noelledaley** [comment]
i had 2 small updates to the instructions to ensure they're accurate, otherwise the code and styling looks great. thanks for doing this!

File: ui/app/templates/components/shamir-modal-flow.hbs (hunk 2 above)
Comment (chelshaw) [reply ui/app/templates/components/shamir-modal-flow.hbs]: Whoops, that was copypasta from the mockups. Good catch!

**Pull request merged**